### DISCUSSION TITLE
Scelta lingua documento

### DISCUSSION THREAD

**Discussion by dmigliacci** [fattureincloud/api#296] [Q&A] [answered]
Buongiorno a tutti,
dato che vendiamo/spediamo anche all'estero, mi domandavo:

Come viene gestita la lingua? In fase di creazione di una proforma/ddt posso scegliere che lingua impostare?
È una scelta a valle?
Dipende dal template?

Grazie in anticipo,
Davide

**Comment by emajo** [comment]
@dmigliacci con lingua si riferisce a quella del documento pdf o ad altro?

**Reply by dmigliacci** [reply]
@emajo a quella utilizzata nel documento pdf, nomi dei campi, intestazioni etc

**Comment by emajo** [comment, marked as answer]
Ciao @dmigliacci, per modificare la lingua del pdf è sufficente passare il campo `language.code` in creazione di documento, valorizzandolo con il codice della lingua voluta, puoi ottenere tutti i code disponibili abbiamo a disposizione l'apposita chiamata [list languages](https://developers.fattureincloud.it/api-reference#get-/info/languages).

Grazie,
Emanuele.

**Reply by dmigliacci** [reply]
Meraviglioso! Queste api sono una bomba, complimenti ragazzi.

Provo subito

**Reply by valmoz** [reply]
Buongiorno @dmigliacci possiamo chiudere la conversazione?
Grazie
